doc/main/intro/limitations.rst

```diff
@@ -44,3 +44,10 @@ Incorrect Installation Location
 **Limitation:** oneTBB does not support ``fork()``. 
 
 **Solution:** To work-around the issue, consider using ``task_scheduler_handle`` to join oneTBB worker threads before using ``fork()``.
+
+Custom Allocator and Topology API Incompatibilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Limitation:** On Linux* OS, using custom memory allocators together with ``tbb::info`` and ``tbb::task_arena::constraints`` APIs may result in undefined behavior.
+
+**Solution:** Set ``TBB_ENABLE_SANITIZERS=1`` in the environment to hint that custom allocators are used.
```

Comment:
It does not seem like the `TBB_ENABLE_SANITIZERS` variable is about usage of custom dynamic malloc replacements and also since the deepbind would be dropped, can we guarantee that the tbbbind will work correct in all use-cases? 
Does it makes sense to provide `LD_PRELOAD`-based workaround instead?

Reply:
I agree that the environment variable name seems unconnected. However, the `LD_PRELOAD`-based workaround might cause more issues due to incorrect/incompatible TBBBind/HWLOC versions.

When deepbind is dropped, I believe we would remove the known limitation and the environment variable, because the globally-defined malloc replacement would always be used.